services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_insights/index.tsx

```diff
@@ -21,7 +34,12 @@ const DiagnosticInsights = ({ classrooms, needsEmailVerification, }) => {
 
   useHideFooter()
 
+  // admins who were created prior to the advent of the admin approval process do not have a verification value, but are treated as approved throughout the codebase
+  const showLockedPage = needsEmailVerification || (role === TEACHER && teacherVerificationStatus !== APPROVED) || (role === ADMIN && adminVerificationStatus && adminVerificationStatus !== APPROVED)
```

Comment:
Beyond the scope of this PR obviously, but I wonder if we should move the abstraction that handles the "these admins are implicitly approved due to age" into the back-end as some sort of attribute so that we don't have to keep checking this combination of "is the admin verified" AND "does the admin NEED to be verified" in a bunch of places.